opm/autodiff/MissingFeatures.cpp

```diff
@@ -66,6 +84,33 @@ namespace MissingFeatures {
                     + "In file " + keyword.getFileName() + ", line " + std::to_string(keyword.getLineNumber()) + "\n";
                 OpmLog::error(msg);
             }
+
+            // check for partially supported keywords.
+            std::multimap<std::string, PartiallySupported<std::string>>::iterator string_it, string_low, string_up;
```

Comment:
It should be possible to write this scan function as a template over the item type:

```C++
template <typename T>
void checkDeck( const Deck& deck , std::multimap<std::string , PartiallySupported<T>>) {
    ....
}
```